editor/src/components/editor/store/editor-state.ts

```diff
@@ -156,7 +156,7 @@ export const defaultUserState: UserState = {
   shortcutConfig: {},
 }
 
-export interface EditorStore {
+export type EditorStore = {
```

Comment:
Why the change from `interface` to `type` here?

Reply:
This is a bit unclear to me.

There was a compile error that went like this:
```
Type 'EditorStore' does not satisfy the constraint 'Record<string | number | symbol, unknown>'.
  Index signature is missing in type 'EditorStore'.ts
```
Changing EditorStore from interface to type fixed this error. I don't quite know why!
The change itself means EditorStore is now a type alias to the `typeof` of the object literal type. 
Maybe that means it also gets some object-like features like an index signature?